src/main/scala/com/databricks/labs/overwatch/validation/DeploymentValidation.scala

```diff
@@ -780,4 +784,32 @@ object DeploymentValidation extends SparkSessionWrapper {
     validationStatus.toArray
   }
 
+  private def validateSystemTableAudit(auditlogprefix_source_path: Option[String], workspace_id: String): DeploymentValidationReport = {
+    val testDetails = s"Testing for System table - ${auditlogprefix_source_path.get}"
+    val ifDataExists = checkSystemTableAudit(auditlogprefix_source_path, workspace_id)
+    if(spark.catalog.tableExists(auditlogprefix_source_path.get) && !ifDataExists) {
```

Comment:
Review this boolean logic, this should be a success if ifDataExists is TRUE

Reply:
`checkSystemTableAudit` is returning `df.isempty` thats why we have applied this logic in the if statement